<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wpointmass</class>
 <widget class="QWidget" name="wpointmass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="GBBodyInformation">
        <property name="title">
         <string>Body information</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="LGeometry">
             <property name="text">
              <string>Geometry</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="LEGeometry">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="PBConnectGeometry">
             <property name="minimumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="LName">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="LEName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="LMass">
             <property name="text">
              <string>Mass</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="LEMass"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="LPosition">
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="LEPosition"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="LEulerParameters">
             <property name="text">
              <string>Euler parameters</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="LEEuerParameters"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="LEulerAngle">
             <property name="text">
              <string>Euler Angle</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QLineEdit" name="LEEulerAngle"/>
           </item>
           <item row="6" column="0" colspan="3">
            <widget class="QGroupBox" name="GBInertia">
             <property name="title">
              <string>Moment of Inertia</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="LIxx">
                <property name="text">
                 <string>Ixx</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="LEIxx"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="LIxy">
                <property name="text">
                 <string>Ixy</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="LEIxy"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="LIyy">
                <property name="text">
                 <string>Iyy</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="LEIyy"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="LIyz">
                <property name="text">
                 <string>Iyz</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="LEIyz"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="LIzz">
                <property name="text">
                 <string>Izz</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="LEIzz"/>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="LIzx">
                <property name="text">
                 <string>Izx</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="LEIzx"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
